src/core/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -294,34 +272,37 @@
 
         </StackPanel>
 
-        <StackPanel x:Name="SidePanel"
-                    Orientation="Vertical"
+        <RelativePanel x:Name="SidePanel" 
                     HorizontalAlignment="Left"
                     Width="{StaticResource SidePanelWidth}"
                     Grid.Column="1">
-
-            <TextBlock x:Uid="About_FancyZones"
+            <StackPanel x:Name="DescriptionPanel">
+                <TextBlock x:Uid="About_FancyZones" x:Name="AboutTitle" Grid.ColumnSpan="2"
                        Style="{StaticResource SettingsGroupTitleStyle}"
                        Margin="{StaticResource XSmallBottomMargin}"/>
+                <TextBlock x:Uid="FancyZones_Description"
+                           TextWrapping="Wrap"
+                           Grid.Row="1" />
```

Comment:
just a quick follow up. Does Grid.Row apply for StackPanel? Just curious!!!

Reply:
Good catch, will fix.